### PR TITLE
Issue-10 Signature Refactoring completed

### PR DESCRIPTION
**Pull request by wolfpack94**
Completed signature refactoring and verified code still works.

### CODE REVIEW

**Comment by j0j0r0** [comment]
Can you please remove the .DS_Store file?

**Comment by wolfpack94** [comment]
Done

**Comment by j0j0r0** [comment]
Ok. I'll complete review hope today. 
Thanks.

File: blockchain/qry.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def format_block_verification(block_verification):
         "verification_id":      block_verification["verification_id"],
         "verified_ts":          format_time(block_verification["verified_ts"]),
         "block_id":             block_verification["block_id"],
-        "signature":            block_verification["signature"],
```
Comment (j0j0r0) [inline blockchain/qry.py]: Can you fix alignment?

Comment (micylt) [inline blockchain/qry.py]: I reviewed the refactor. Looks good to me.

**Pull request merged**